internal/services/cognitive/cognitive_account_resource.go

```diff
@@ -70,6 +70,7 @@ func resourceCognitiveAccount() *pluginsdk.Resource {
 				ForceNew: true,
 				ValidateFunc: validation.StringInSlice([]string{
 					"Academic",
+					"AIServices",
```

Comment:
This Resource is a bit of a misnomer, in general `kind` is used to provision a unique Resource Type within Azure - given there's AI specific behaviours, it'd be better for this to become a new resource in the long-term (with AI specific fields/validation) rather than shoe-horning this into the existing resource - even though this'll make things easier in the short-term.

This is particularly true because some of the values for `kind` have been removed over the years due to breaking changes in the API - as such I think it'd probably be worth creating an `azurerm_cognitive_ai_services_account` resource here instead - WDYT?

Reply:
@xuzhang3 in this instance we should split this out rather than add a new `kind` value here, so I don't think we should support this in the existing resource unfortunately - since the properties for this new "AI" kind are likely to be different than the other kind's/cause issues down the line